packages/protocol/test-sol/constants.sol

```diff
@@ -6,10 +6,9 @@ contract TestConstants {
   uint256 public constant FIXED1 = 1e24;
   uint256 public constant MINUTE = 60;
   uint256 public constant HOUR = 60 * MINUTE;
-  uint256 public constant DAY = 24 * HOUR;
-  uint256 public constant MONTH = 30 * DAY;
-  uint256 constant WEEK = 7 * DAY;
-  uint256 public constant YEAR = 365 * DAY;
+  uint256 public constant MONTH = 30 * 86400;
+  uint256 constant WEEK = 7 * 86400;
+  uint256 public constant YEAR = 365 * 86400;
```

Comment:
Actually, shouldn't it be the previous version?

Reply:
The `DAY` variable is already declared in a different contract and was causing some inheritance conflict. Used a different name in this contract to avoid magic numbers.